src/docs/development/add-to-app/android/add-flutter-screen.md

```diff
@@ -157,10 +157,24 @@ Now, when you launch `FlutterActivity`, there is significantly less delay in
 the display of Flutter content.
 
 {{site.alert.note}}
-  When re-using a cached engine across multiple `FlutterActivity` instances, you 
-  assume responsibility for showing the desired content for the given `Activity`.
-   Strategies for switching content in a shared `FlutterEngine` are beyond the 
-   scope of this guide. TODO(mattcarroll): link to resource.
+  When using a cached `FlutterEngine`, that `FlutterEngine` outlives any
+  `FlutterActivity` or `FlutterFragment` that displays it. You should keep in
+  mind that Dart code begins executing as soon as you pre-warm the 
+  `FlutterEngine`, and continues executing after the destruction of your
+  `FlutterActivity`/`FlutterFragment`. To stop executing and clear resources,
+  obtain your `FlutterEngine` from the `FlutterEngineCache` and destroy the
+  `FlutterEngine` with `FlutterEngine.destroy()`.
+{{site.alert.end}}
+
+{{site.alert.note}}
+  Runtime performance is not the only reason that one might pre-warm and cache
+  a `FlutterEngine`. A pre-warmed `FlutterEngine` executes Dart code independent
+  from a `FlutterActivity`, which allows such a `FlutterEngine` to be used to
+  execute arbitrary Dart code at any moment in time. Non-UI application logic
+  can be executed in a `FlutterEngine`, like networking and data caching, as 
+  well as background behavior within a `Service` or elsewhere. When using a
+  `FlutterEngine` to execute behavior in the background, be sure to accomodate
```

Comment:
accomodate => accommodate

Is accommodate the best word here? Do you mean "follow" or "honor"? I'm not sure what you mean by "accommodating" a restriction.